benchmarks/distributed/rpc/rl/report.json

```diff
@@ -0,0 +1 @@
+{"world_size": 5, "master_addr": "127.0.0.1", "master_port": "29501", "state_size": "10,20,10", "nlayers": 5, "out_features": 10, "graph_variable": "batch", "x_axis_name": "batch", "benchmark_results": [{"agent latency": {"50": 0.003, "75": 0.003, "90": 0.003, "95": 0.003}, "agent throughput": {"50": 1180, "75": 1211, "90": 1239, "95": 1256}, "observer latency": {"50": 0.003, "75": 0.003, "90": 0.003, "95": 0.004}, "observer throughput": {"50": 307, "75": 315, "90": 320, "95": 323}, "batch": true}, {"agent latency": {"50": 0.006, "75": 0.006, "90": 0.006, "95": 0.006}, "agent throughput": {"50": 175, "75": 182, "90": 190, "95": 194}, "observer latency": {"50": 0.007, "75": 0.007, "90": 0.007, "95": 0.007}, "observer throughput": {"50": 151, "75": 157, "90": 162, "95": 167}, "batch": false}]}
```

Comment:
@mrshenli please let me know your thoughts on this as an output of the benchmark graphing run for the plot repo to use as an input to plot graphs from